libsolidity/ast/ASTJsonConverter.cpp

```diff
@@ -268,7 +268,7 @@ bool ASTJsonConverter::visit(InheritanceSpecifier const& _node)
 {
 	setJsonNode(_node, "InheritanceSpecifier", {
 		make_pair("baseName", toJson(_node.name())),
-		make_pair("arguments", _node.arguments() ? toJson(*_node.arguments()) : Json::Value(Json::arrayValue))
+		make_pair("arguments", _node.arguments() ? toJson(*_node.arguments()) : Json::nullValue)
```

Comment:
I also changed ``Json::nullValue`` in the absence of arguments here, although this should actually have been done in #3824.

Reply:
Great, thanks!